bridge-to-the-moon/package-lock.json

```diff
@@ -0,0 +1,3701 @@
+{
```

Comment:
This is really horrible :-(
Is there a different package that does not have 10000 dependencies?

Reply:
ether.js maybe?
Or perhaps going to python?

Reply:
@chriseth which package in particular has too many dependencies?